VideoClassification/reader_utils.py

```diff
@@ -0,0 +1,55 @@
+import readers
+import tensorflow as tf
+from tensorflow.io import gfile
+
+
+def get_reader(feature_names='rgb,audio', feature_sizes='1024,128'):
+	# Convert feature_names and feature_sizes to lists of values.
+	feature_names, feature_sizes = GetListOfFeatureNamesAndSizes(feature_names, feature_sizes)
+
+	reader = readers.YT8MFrameFeatureDataset(feature_names=feature_names, feature_sizes=feature_sizes)
+
+	return reader
+
+def GetListOfFeatureNamesAndSizes(feature_names, feature_sizes):
+	"""Extract the list of feature names and the dimensionality of each feature
+
+		 from string of comma separated values.
+
+	Args:
+		feature_names: string containing comma separated list of feature names
+		feature_sizes: string containing comma separated list of feature sizes
```

Comment:
Should these two have same lengths?

Reply:
yes they should. I will add a comment in the docstring